crawler/mergeData.py

```diff
@@ -0,0 +1,51 @@
+import json
+
+with open('yelpData.json', 'r') as yelpfile:
+    yelpdata = json.load(yelpfile)
+
+with open('googleData.json', 'r') as googlefile:
+    googledata = json.load(googlefile)
+
+def checkDuplicated(yelpAddress, googleAddress):
+    yelpAddress_split = []
+    googleAddress_spilt = []
+
+    yelpAddress_split = yelpAddress.split(' ')
+    googleAddress_split = googleAddress.split(' ')
+
+    yelpAddress_part = yelpAddress_split[0] + " " + yelpAddress_split[1]
+    googleAddress_part = googleAddress_split[0] + " " + googleAddress_split[1]
+
+    if(yelpAddress_part == googleAddress_part):
+        return True
+    else:
+        return False
+
+
+# main
+
+for googleitem in googledata:
+    googleAddress = googleitem['address']
+    for yelpitem in yelpdata:
+        yelpAddress = yelpitem['address']
+        if(checkDuplicated(yelpAddress, googleAddress)):
+            if not (yelpitem.get('phone')) : 
+                if(googleitem.get('phone')):
+                    yelpitem['phone'] = googleitem['phone']
```

Comment:
Just trying to understand the logic here: why are we assigning googleitem['phone'] to yelpitem['phone']?